src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -200,11 +208,13 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
 
   useEffect(() => {
     if (isSubmitted) {
-      document.querySelector(".processor-field-error")?.scrollIntoView({
-        behavior: "smooth",
-        block: "start",
-        inline: "nearest",
-      });
+      document
+        .querySelector(".processor-field-error, .pf-m-error")
+        ?.scrollIntoView({
```

Comment:
I wasn't able to find a way to add a custom css class to invalid fields generated by uniforms-patternfly, So here I am targeting the PF error class to scroll invalid fields into view.